tests/gold_tests/pluginTest/prefetch_simple/prefetch_simple.test.py

```diff
@@ -0,0 +1,76 @@
+'''
+'''
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+Test.Summary = '''
+Test prefetch.so plugin (simple mode).
+'''
+
+server = Test.MakeOriginServer("server")
+for i in range(4):
+    request_header = {
+        "headers":
+            f"GET /texts/demo-{i + 1}.txt HTTP/1.1\r\n"
+            "Host: does.not.matter\r\n"  # But cannot be omitted.
+            "\r\n",
+            "timestamp": "1469733493.993",
+            "body": ""
+    }
+    response_header = {
+        "headers":
+            "HTTP/1.1 200 OK\r\n"
+            "Connection: close\r\n"
+            "Cache-control: max-age=85000\r\n"
+            "\r\n",
+        "timestamp": "1469733493.993",
+        "body": f"This is the body for demo-{i + 1}.txt.\n"
+    }
+    server.addResponse("sessionlog.json", request_header, response_header)
+
+server.Streams.stdout = "server.gold"
+
+dns = Test.MakeDNServer("dns")
+
+ts = Test.MakeATSProcess("ts", block_for_debug=False)
+ts.Disk.records_config.update({
+    'proxy.config.diags.debug.enabled': 1,
+    'proxy.config.diags.debug.tags': 'http|dns|prefetch',
+    'proxy.config.dns.nameservers': f"127.0.0.1:{dns.Variables.Port}",
+    'proxy.config.dns.resolv_conf': "NULL",
+})
+ts.Disk.remap_config.AddLine(
+    f"map http://domain.in http://127.0.0.1:{server.Variables.Port}" +
+    " @plugin=cachekey.so @pparam=--remove-all-params=true"
+    " @plugin=prefetch.so" +
+    " @pparam=--front=true" +
```

Comment:
Why do we need this?  The test fails without this parameter.

Reply:
Is your question really, "why does the prefetch plugin require this parameter instead of setting it by default?"

Reply:
Could be, I'm very unfamiliar with this plugin, I don't think anyone in Yahoo uses it.  I thought t he "simple" set up means single tier, and that "front" means first in tier in a dual tier setup, so not relevant.